transport-msg.tex

```diff
@@ -612,29 +602,39 @@ \subsubsection{Driver Notifications}
         information, such as the next descriptor offset or wrap counter
```

Comment:
There is a feature bit which is configuring if or not those information are available (VIRTIO_F_NOTIFICATION_DATA). Maybe we should mention that those fields are only set if this has been negociated.

Reply:
I did elsewhere.  We can add it again here or reduce the redundancy